ui_testing/testcases/basic_tests/test004_testunits.py

```diff
@@ -1362,3 +1362,23 @@ def test050_sub_and_super_scripts_in_active_table(self) :
         span = cells['Unit'].find_element_by_class_name('white-tooltip')
         required_value = span.get_attribute('ng-reflect-ngb-tooltip')
         self.assertEqual(payload['unit'],required_value)
+
+    def test051_cancel_testunit_name_configuration(self):
+        """
+        Test unit: Configuration: make sure that when you select from test unit name configuration
+        drop down list and press on cancel button, nothing changed
+        LIMS-5650
+        """
+        self.info('open test unit configurations')
+        self.test_unit_page.open_configurations()
+        self.info('open test unit name configuration options')
+        self.test_unit_page.open_testunit_name_configurations_options()
+        self.info('get selected value in view& search field')
+        selected_option = self.base_selenium.get_text(element='configurations_page:selected_option').replace('×', '')
+        self.info('select another option and press cancel')
+        self.base_selenium.select_item_from_drop_down(element='configurations_page:view_search_ddl')
+        self.base_selenium.click(element='configurations_page:configuration_popup_cancel_btn')
```

Comment:
create `set_selected_value_searc_field(save=True)` method in test_unit page and use it here

Reply:
done